lib/utils/pulsar_functions.h

```diff
@@ -0,0 +1,41 @@
+#ifndef PULSAR_FUNCTIONS_H
+#define PULSAR_FUNCTIONS_H
+
+#include <inttypes.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Also I don't think we need this (and the 3 corresponding lines below) anymore either.  This was a leftover from the days when the network capture code was written in C not C++.  If the code compiles without it, then just leave it out.  And we can change the name of this file to `pulsar_functions.hpp`

Reply:
sounds good; i removed both of the `#ifdef _cplusplus_` lines, renamed the file and pushed.

Reply:
ah yes, going to change the include call, one moment